src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/Nodes/NodeTypes.cs

```diff
@@ -1,147 +1,143 @@
-﻿//
+
```

Comment:
There generally shouldn't be a blank line before the copyright, remove line 12 from the template